CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 3.2.0
+* Added `PolicyMachine#accessible_objects_for_operations` for finding a map of accessible
```

Comment:
Thanks for using PolicyMachine language (object) to describe these concepts rather than Dalton language (operable).